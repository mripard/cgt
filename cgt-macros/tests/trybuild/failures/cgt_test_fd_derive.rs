fn main () {
    cgt_macros::cgt_test_with_fd!();
}