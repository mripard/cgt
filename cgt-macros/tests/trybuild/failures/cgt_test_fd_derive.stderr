error: expected macro, found attribute macro `cgt_macros::cgt_test_with_fd`
 --> tests/trybuild/failures/cgt_test_fd_derive.rs:2:5
  |
2 |     cgt_macros::cgt_test_with_fd!();
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not a macro
